cfpb/capital-framework
# Update Featured Content Module styling

This PR implements several desired styling updates for Featured Content Modules. See https://GHE/CFGOV/platform/issues/3087 for background.

/cc @designlanguage @keelerr

## Removals

- Remove category headers from FCM templates

## Changes

- Change FCM min height from 320px to 220px
- Set FCM image width to 270px, let text flex
- Change FCM breakpoint to 600px
- Add bottom padding to FCM image on XS screens
- Sets border on the FCM itself, instead of surrounding block
  - The border is inherent to the FCM styling and shouldn't be set on the block. It makes even less sense now that the block modifier for adding the negative side margins is no longer used.
- Updates docs

## Testing

1. Check out a working branch
1. Run `gulp docs`
1. Go into the documentation directory with `cd docs`
1. Run `bundle exec jekyll serve` and visit http://localhost:4000/components/cf-layout/#featured-content-module
1. Go into responsive mode and view the FCM at various window widths

## Screenshots

### Before

![Screen Shot 2019-06-06 at 13 12 40](https://user-images.githubusercontent.com/1044670/59052375-fc228400-885c-11e9-8f16-428041517f46.png)

### After

![Screen Shot 2019-06-06 at 13 13 33](https://user-images.githubusercontent.com/1044670/59052395-047abf00-885d-11e9-9005-d5a6290cc6ef.png)

## Todos

- [ ] Open corresponding PR in cfgov-refresh

## Checklist

- [x] PR has been tagged with the type of change it includes
      (breaking, enhancement, bug, documentation or internal)
- [x] Project documentation has been updated in this PR with the `gulp docs` command

## Testing checklist

### Browsers


### Accessibility


### Other
